.github/workflows/semantic-pull-request.yaml

```diff
@@ -0,0 +1,14 @@
+name: semantic-pull-request
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+jobs:
+  semantic-pull-request:
+    uses: tier4/autoware-github-actions/.github/workflows/reusable-semantic-pull-request.yaml@main
```

Comment:
Might be moved to autowarefoundation.

Reply:
https://github.com/autowarefoundation/autoware-github-actions/blob/e354248417106605f3a54d14717646347419a85d/.github/workflows/reusable-semantic-pull-request.yaml